packages/pds/src/api/com/atproto/server/createAccount.ts

```diff
@@ -132,6 +120,10 @@ export default function (server: Server, ctx: AppContext) {
             .execute()
         }
 
+        if (!entrywayAssignedPds) {
+          await repoTxn.createRepo(did, [], now)
+        }
```

Comment:
I'm not sure how important this case actually is for the entryway, which shouldn't end-up hosting any repositories.  But if we create a repository and sequence them onto the stream prior to submitting their plc op, it's possible their first event wont get processed by a consumer.

Reply:
yup i thought about that

but i also think we just want to remove all this repo logic out of the entryway anyway & it's not currently in use so we don't really run a risk with it